apps/kindergarten/views.py

```diff
@@ -1 +1,19 @@
-# Create your views here.
+from django.shortcuts import render_to_response
+from apps.kindergarten.models import (
+    Kindergarten,
+    Group
+)
+from apps.kindergarten.templates import *
+
+from django.views.generic.detail import DetailView
+
+class KinderView(DetailView):
+    template_name = 'show_kindergarten.html'
```

Comment:
move this template to kindergarten folder (in 'templates' folder)

Reply:
sorry, i don't understand what and where to move

Reply:
show_kindergatren is in apps/kindergarten/templates, no?